codec-registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/ClassSweepCompilerPluginComponent.scala

```diff
@@ -15,8 +15,8 @@ class ClassSweepCompilerPluginComponent(options: CodecRegistrationCheckerOptions
   override val runsAfter: List[String] = List("refchecks")
   override def description: String = s"searches for direct descendants of classes annotated with serializability trait"
 
-  val foundTypes: mutable.Buffer[(String, String)] = mutable.ListBuffer()
-  val typesToUpdate: mutable.Buffer[(String, String)] = mutable.ListBuffer()
+  val foundTypes: mutable.Set[(String, String)] = mutable.Set()
+  val typesToUpdate: mutable.Set[(String, String)] = mutable.Set()
```

Comment:
`ClassSweep` phase did not have performance problems, so it's natural that optimisations here don't have any effects.

Furthermore this optimisation may be a downgrade. `foundTypes` and `typesToUpdate` are only being appended to, something buffers are made for. But `HashSet` starts with `defaultInitialCapacity` of 16, meaning that multiple resizes are required during runtime.

Reply:
I changed that because in the next phase there are `foundTypes.toSet` and `typesToUpdate.toSet`.

Reply:
please revert it